spark/src/main/scala/org/apache/spark/ml/classification/SmileClassifier.scala

```diff
@@ -0,0 +1,238 @@
+package org.apache.spark.ml.classification
```

Comment:
I wouldn't pollute spark's namespace.

Reply:
yes but there is a lot of useful code which is private spark I use to make the code more robust/production grade, do you prefer that i copy paste what I need from spark ? I have used the same approach as : https://github.com/Azure/mmlspark for namespaces